tests/unit/store/auth.module.spec.js

```diff
@@ -0,0 +1,143 @@
+import sinon from "sinon";
+
+import { mutations, actions, getters } from "@/store/auth.module";
+import { SET_AUTH, PURGE_AUTH, SET_ERROR } from "@/store/mutations.type";
+import JwtService from "@/common/jwt.service";
+import ApiService from "@/common/api.service";
+import {
+  LOGIN,
+  LOGOUT,
+  REGISTER,
+  CHECK_AUTH,
+  UPDATE_USER
+} from "@/store/actions.type";
+
+describe("getters", () => {
+  const state = {
+    user: "Test User",
+    isAuthenticated: true
+  };
+
+  it("currentUser", () => {
+    expect(getters.currentUser(state)).toBe("Test User");
+  });
+
+  it("isAuthenticated", () => {
+    expect(getters.isAuthenticated(state)).toBe(true);
+  });
+});
+
+describe("mutations", () => {
+  it("SET_ERROR", () => {
+    const setError = mutations[SET_ERROR];
+    const state = { errors: null };
+    setError(state, "Test Error");
+
+    expect(state.errors).toBe("Test Error");
+  });
+
+  it("SET_AUTH", () => {
+    const stub = sinon.stub(JwtService, "saveToken");
```

Comment:
We do not need `sinon` here. Jest is capable of mocking JavaScript.

**Resources**

- [ES6 Class Mocks](https://jestjs.io/docs/en/es6-class-mocks)
- [Manual Mocks](https://jestjs.io/docs/en/manual-mocks)
- [Mock Functions](https://jestjs.io/docs/en/mock-functions)

Reply:
Good point, too used to the mocha+sinon combo